pandas/tests/io/test_gcs.py

```diff
@@ -60,6 +61,29 @@ def open(*args):
     assert_frame_equal(df1, df2)
 
 
+@td.skip_if_no("gcsfs")
+def test_to_parquet_gcs_new_file(monkeypatch, tmpdir):
```

Comment:
Looks reasonable.

You'll need to skip if fastparquet isn't installed.

Reply:
done